virtool/users/sessions.py

```diff
@@ -29,9 +33,7 @@ async def create_session(
         expires_at = utc.shift(minutes=10)
 
     new_session = {
-        "_id": session_id,
-        "created_at": virtool.utils.timestamp(),
-        "expiresAt": expires_at.datetime,
+        "created_at": virtool.utils.timestamp().timestamp(),
```

Comment:
This works?

Reply:
Yeah, kinda ugly, but the first timestamp call is to the `virtool.utils.timestamp()` which returns a datetime object, while the second is a call to `datetime.timestamp()`. The second call converts the datetime object to POSIX float timestamp that is compatible with the limitations of redis's allowed datatypes and json's decoding. I could sidestep this a bit by calling `arrow.utcnow().naive` right in this function, but it doesn't make things better imo.